controllers/util.go

```diff
@@ -0,0 +1,74 @@
+package controllers
+
+import (
+	"slices"
+
+	synv1alpha1 "github.com/projectsyn/lieutenant-operator/api/v1alpha1"
+	v1 "k8s.io/api/core/v1"
+)
+
+func envVarIndex(name string, list *[]synv1alpha1.EnvVar) int {
```

Comment:
Why do we have a pointer to the list here? It's the function that needs it the least 😅

Reply:
🙃